src/components/ColorPicker/ColorPickerItem/ColorPickerItem.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import styles from './ColorPickerItem.st.css';
+import { RadioButton } from 'wix-ui-core/radio-button';
+import { RadioButtonProps } from 'wix-ui-core/dist/src/components/radio-button/RadioButton';
+
+export interface ColorPickerItemProps extends RadioButtonProps {
+  className?: string;
+  key?: number;
+}
+
+/** ColorPickerItem */
+export class ColorPickerItem extends React.Component<ColorPickerItemProps> {
+  static displayName = 'ColorPickerItem';
+
+  getRadio = ({ value }: ColorPickerItemProps) => (
+    <div className={styles.radioOuter}>
+      <div className={styles.radioInner} style={{ backgroundColor: value }} />
+    </div>
+  );
+
+  render() {
+    const { value, checked, onChange } = this.props;
+
+    return (
+      <RadioButton
+        {...styles('root', { checked }, { ...this.props })}
+        aria-label={this.props['aria-label']}
+        data-hook={ColorPickerItem.displayName}
```

Comment:
And then here, instead of using the displayName, you use the datahook defined in the dataHook file

Reply:
now displayName equals dataHook, so I just made a single source of truth in dataHook file